armi/bookkeeping/db/tests/test_database3.py

```diff
@@ -226,6 +224,37 @@ def makeHistory(self):
 
             self.db.writeToDB(self.r)
 
+    def test_loadOperator(self):
+        self.makeHistory()
+        self.db.close()
+        # Write a bad setting
+        with h5py.File(self.db.fileName, "r+") as hf:
+            settings = hf["inputs/settings"].asstr()[()]
+            settings += "  fakeSetting: I'll be back"
+            del hf["inputs/settings"]
+            hf["inputs/settings"] = settings
```

Comment:
ok so I didn't have time to dig in on how to save new settings to the DB, but that's the intent here. Once that is done I bet the test will work!

Reply:
once this test works I might be able to add more

Reply:
yay figured it out! pushing up new tests soon